src/main/java/cc/polyfrost/oneconfig/utils/Notifications.java

```diff
@@ -245,4 +249,27 @@ else if (desiredPosition != entry.getValue().getEnd())
         });
         deltaTime = 0;
     }
+
+    private Animation dummyAnimation;
+    private static final Icon DEFAULT_ICON = new Icon(SVGs.ONECONFIG_HEAD_DARK);
+
+    @Subscribe
+    private void onTickEvent(TickEvent event) {
+        if (Platform.getServerPlatform().doesPlayerExist() && event.stage == Stage.START) {
```

Comment:
first launch check here so it is optimized out quicker

Reply:
> first launch check here so it is optimized out quicker

its literally a null check and an equals check